vmm/src/vm.rs

```diff
@@ -213,9 +213,20 @@ fn get_host_cpu_phys_bits() -> u8 {
     unsafe {
         let leaf = x86_64::__cpuid(0x8000_0000);
 
+        let reduced = if leaf.eax >= 0x8000001f
```

Comment:
I think this needs some comments (perhaps referencing a spec.)

Reply:
ok. I will add them.